src/App.js

```diff
@@ -1,23 +1,61 @@
-import logo from './logo.svg';
+
+
+
+import {useState, useEffect} from "react";
+import axios from "axios" ;
```

Comment:
I suggest removing line 5 if you are not using axios for your fetch request. What do you think?